draft-ietf-dhc-rfc8415bis-01.xml

```diff
@@ -3631,21 +3622,14 @@
         <name>Reception of Unicast Messages</name>
         <t>The server is not supposed to accept unicast traffic from a
         client. The Server Unicast option (see <xref target="RFC3315-22.12" format="default"/>)
-        has been deprecated and hence clients should no longer send the
-        server unicast messages. However, a server that may have supported
-        the Server Unicast option and is upgraded to not support it, may receive a
-        unicast message if it previously sent the client the Server Unicast
-        Option.</t>
-        <t>When the server receives a message via unicast from a client,
-        the server discards that message and responds with an Advertise
-        (when responding to a Solicit message) or Reply message (when
-        responding to any other messages) containing a Status Code option
-        (see <xref target="RFC3315-22.13" format="default"/>) with the
-        value UseMulticast, a Server Identifier option
-        (see <xref target="RFC3315-22.3" format="default"/>) containing the
-        server's DUID, the Client Identifier option
-        (see <xref target="RFC3315-22.2" format="default"/>) from the client
-        message (if any), and no other options.</t>
+        and UseMulticast status code (see <xref target="RFC3315-22.13" format="default"/>)
+        have been obsoleted and hence clients should no longer send the
+        server unicast messages nor receive the UseMulticast status code.
```

Comment:
I don't like the "server unicast messages". It's ambiguous. Is it "messages with server-unicast option", "messages sent to the server unicast address" or both? We should rephrase. Maybe "messages sent to the server unicast address"?

Reply:
```suggestion
        messages sent to the server unicast address nor receive the UseMulticast status code.
```

Reply:
This text would be broken because of the "the" on line 3626 ("send the messages sent to the server unicast address"). I'm not sure how to adjust the text to fix this in GitHub tool here? I think we should say "and hence clients should no longer send messages to a server's unicast address nor receive the UseMulticast status code."